AUTHORS

```diff
@@ -1,3 +1,6 @@
 Timur Shemsedinov <timur.shemsedinov@gmail.com>
+Dimon Durak <Dimon Durak>
+Artem Chernenkiy <notthewhite@gmail.com>
 Alexey Orlenko <eaglexrlnk@gmail.com>
 Mykola Bilochub <nbelochub@gmail.com>
+Vlad <Dzyubavlad@gmail.com>
```

Comment:
@DzyubSpirit please check if you email is correct here. And how would you prefer your full name to be spelled?